crates/ruff_python_parser/src/parser/expression.rs

```diff
@@ -1375,10 +1374,12 @@ impl<'src> Parser<'src> {
     }
 
     /// Parses an expression in parentheses, a tuple expression, or a generator expression.
+    ///
+    /// Matches the `(tuple | group | genexp)` rule in the [Python grammar].
+    ///
+    /// [Python grammar]: https://docs.python.org/3/reference/grammar.html
     fn parse_parenthesized_expression(&mut self) -> ParsedExpr {
         let start = self.node_start();
-        let saved_context = self.set_ctx(ParserCtxFlags::PARENTHESIZED_EXPR);
-
```

Comment:
This isn't being used anywhere.

Reply:
Can we remove `ParserCtxFlags::PARENTHESIZED_EXPR` too or is that used somehwere? If so, is it just that this rule doesn't exercise the code path where `PARENTHESIZED_EXPR` is used?

Reply:
Yes, I'll remove it. I missed that.